sympy/concrete/summations.py

```diff
@@ -1137,9 +1137,16 @@ def eval_sum_symbolic(f, limits):
         if without_i != 0:
             s = eval_sum_symbolic(with_i, (i, a, b))
             if s:
-                r = without_i*(b - a + 1) + s
-                if r is not S.NaN:
-                    return r
+                if((s == S.NegativeInfinity and (b - a + 1) == S.Infinity) or (s == S.Infinity and (b - a + 1) == S.Infinity)):
+                    Max = max(abs(f.subs(i, a)), abs(f.subs(i, S.Infinity)))
```

Comment:
Slightly better to use the Sympy `Max` and `Abs` directly (and therefore rename the variable).

Reply:
I used `max` and `abs` to maintain consistency in the code , all throughout summations.py these ones have been used everywhere . Even the is_absolutely_convergent() method which just a a single line uses `abs` .

Reply:
OK! It will work, in the sense that Sympy will turn them into `Abs` and `Max` with some additional processing, but probably they should be changed elsewhere as well (although it must be done a bit carefully, sometimes they do compute the `max` and `abs` of standard Python types, like the result of `len`). 

I will probably change it at some stage, so feel free to do that already now...

Reply:
At least, please rename the `Max` variable to make it less confusing.

Reply:
> OK! It will work, in the sense that Sympy will turn them into `Abs` and `Max` with some additional processing, but probably they should be changed elsewhere as well (although it must be done a bit carefully, sometimes they do compute the `max` and `abs` of standard Python types, like the result of `len`).
> 
> I will probably change it at some stage, so feel free to do that already now...

Yeah exactly it's not that I didn't try , my plan earlier was to go with Abs, Max but and I had made relevant changes at a couple of places where I found it necessary for this PR but that was breaking some things from my side hence I decided to switch back .